src/app/state/movies/movies.reducer.ts

```diff
@@ -1,15 +1,20 @@
 import { createReducer, on, Action } from '@ngrx/store';
 import { Movie } from 'src/app/interfaces';
 import { moviesInitialState, MovieState } from './movies-state';
-import { addMovie } from './movies.actions';
+import * as fromMovie from './movies.actions';
 
 const reducer = createReducer(
   moviesInitialState,
-  on(addMovie, (state: MovieState, payload: { movie: Movie }) => ({
+  on(fromMovie.addMovie, (state: MovieState, payload: { movie: Movie }) => ({
     ...state,
     movies: [...state.movies, payload.movie],
-  })
-  ),
+    loading: true,
+  })),
+  on(fromMovie.addMovieSuccess, (state: MovieState, { movie }) => ({
+    ...state,
+    movie,
+    loading: false,
```

Comment:
a tutaj elegancko ustawiasz loading: false na state movies, ale na filmmie dalej jest loading:true i pewnie dlatego w komponencie Ci się nie zmienia. W detoolsach jak przejrzysz filmy to też się nie zmieni, a patrzyłeś pewnie namovies.loading i dlatego myślałeś że się zmieniło